src/main/java/uk/gov/hmcts/reform/idam/health/am/AmIsAliveHealthProbe.java

```diff
@@ -29,11 +33,21 @@ public boolean probe() {
                 log.info(TAG + "success");
                 return true;
             } else {
-                log.error(TAG + "response did not contain expected value");
+                String msg = TAG + "response did not contain expected value";
+                log.error(msg);
```

Comment:
logging could be improved by using "{}" instead of concatenation

Reply:
I prefer the look of concatenation and it's also faster. Then again, the look of it is just my opinion.

Reply:
I could still change it if you feel strongly about it :)